site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -930,13 +931,19 @@ components:
          properties:
             "mode":
                type: string
-               description: The mode of the data usage caluclation. The supported modes by
-                            the default resource limit checks implementation are "days"
-                            and "monthly".
+               description: |
+                 The accounting period mode of recurrence. The following modes are defined:
+                 * `monthly`: Accounting periods start at 12 AM UTC on the 1st day of each month.
```

Comment:
Using `midnight (start of day) UTC` instead of `12 AM UTC` would be clearer and easier to understand here, IMHO. There might be some [confusion](https://en.wikipedia.org/wiki/12-hour_clock#Confusion_at_noon_and_midnight) otherwise.

Reply:
Yes, I guess that will make it easier to understand for people from Europe ...